tools/src/test/java/org/apache/kafka/tools/consumer/group/DescribeConsumerGroupTest.java

```diff
@@ -141,699 +186,857 @@ public void testPrintVersion(String quorum) {
-
-        String[] cgcArgs = new String[]{"--bootstrap-server", bootstrapServers(listenerName()), "--describe", "--group", GROUP};
-        ConsumerGroupCommand.ConsumerGroupService service = getConsumerGroupService(cgcArgs);
-
-        TestUtils.waitForCondition(() -> {
-            Entry<Optional<ConsumerGroupState>, Optional<Collection<MemberAssignmentState>>> res = service.collectGroupMembers(GROUP, false);
-            return res.getKey().map(s -> s.equals(ConsumerGroupState.STABLE)).orElse(false) &&
-                res.getValue().isPresent() &&
-                res.getValue().get().stream().filter(s -> Objects.equals(s.group, GROUP)).count() == 2 &&
-                res.getValue().get().stream().filter(x -> Objects.equals(x.group, GROUP) && x.numPartitions == 1).count() == 2 &&
-                res.getValue().get().stream().noneMatch(x -> Objects.equals(x.group, GROUP) && x.numPartitions == 0);
-        }, "Expected two rows (one row per consumer) in describe group members results.");
+            // run two consumers in the group consuming from a two-partition topic
+            try (AutoCloseable protocolConsumerGroupExecutor = consumerGroupClosable(groupProtocol, group, topic, Collections.emptyMap(), 2);
+                 ConsumerGroupCommand.ConsumerGroupService service = consumerGroupService(new String[]{"--bootstrap-server", clusterInstance.bootstrapServers(), "--describe", "--group", group});
+            ) {
+                TestUtils.waitForCondition(() -> {
+                    GroupState state = service.collectGroupState(group);
+                    return Objects.equals(state.state, ConsumerGroupState.STABLE) && Objects.equals(state.group, group) && state.numMembers == 2;
+                }, "Expected a stable group with two members in describe group state result.");
+            }
+        }
+    }
 
-        Entry<Optional<ConsumerGroupState>, Optional<Collection<MemberAssignmentState>>> res = service.collectGroupMembers(GROUP, true);
-        assertTrue(res.getKey().map(s -> s.equals(ConsumerGroupState.STABLE)).orElse(false) && res.getValue().map(s -> s.stream().filter(x -> x.assignment.isEmpty()).count()).orElse(0L) == 0,
-            "Expected additional columns in verbose version of describe members");
+    @ClusterTemplate("generator")
+    public void testDescribeSimpleConsumerGroup() throws Exception {
+        // Ensure that the offsets of consumers which don't use group management are still displayed
+        for (GroupProtocol groupProtocol: clusterInstance.supportedGroupProtocols()) {
+            String topic = TOPIC_PREFIX + groupProtocol.name();
+            String group = GROUP_PREFIX + groupProtocol.name();
+            createTopic(topic, 2);
+
+            try (AutoCloseable protocolConsumerGroupExecutor = consumerGroupClosable(GroupProtocol.CLASSIC, group, new HashSet<>(Arrays.asList(new TopicPartition(topic, 0), new TopicPartition(topic, 1))), Collections.emptyMap());
```

Comment:
why using `GroupProtocol.CLASSIC` rather than `groupProtocol` ?

Reply:
This follows original test case. 

1. It uses `addSimpleGroupExecutor`.
https://github.com/apache/kafka/blob/eea9ebf8a31126f797a4a325995e5e04d4993b74/tools/src/test/java/org/apache/kafka/tools/consumer/group/DescribeConsumerGroupTest.java#L710

2. For `SimpleGroupExecutor`, it uses default group protocol, so I use CLASSIC here.

https://github.com/apache/kafka/blob/eea9ebf8a31126f797a4a325995e5e04d4993b74/tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTest.java#L164-L168
https://github.com/apache/kafka/blob/eea9ebf8a31126f797a4a325995e5e04d4993b74/tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTest.java#L317-L325